libcommon/include/pivot/utility/entity.hxx

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <cinttypes>
+#include <compare>
+#include <iostream>
+#include <limits>
+
+namespace pivot
+{
+/// @brief Id of an entity in the ECS
+using Entity = std::uint32_t;
+/// Maximum number of entity in the ECS
+const Entity MAX_ENTITIES = 500000;
```

Comment:
Do we really need this value ? It was used to mark an invalid id of an entity, but now that we have null entity, might be worth to remove this. But out of the subjet here...

Reply:
Looks like it is still used